profitbricks/src/main/java/org/jclouds/profitbricks/compute/internal/ProvisioningStatusPollingPredicate.java

```diff
@@ -50,6 +50,8 @@ public boolean apply(String input) {
             return expect == api.dataCenterApi().getDataCenterState(input);
          case SERVER:
             return expect == api.serverApi().getServer( input ).state();
+         case STORAGE:
+            return expect == api.storageApi().getStorage( input ).state();
```

Comment:
Add the server and storage cases to the `ProvisioningStatusPollingPredicateTest`.